welcome_wizard/banner.py

```diff
@@ -0,0 +1,32 @@
+"""Home page banner for Welcome Wizard."""
+from typing import Optional
+
+from django.urls import reverse
+from django.utils.html import format_html
+
+from nautobot.extras.choices import BannerClassChoices
+from nautobot.extras.plugins import PluginBanner
+
+
+def banner(context, *args, **kwargs) -> Optional[PluginBanner]:
+    """
+    Construct the Welcome Wizard Banner on the Home Page.
+
+    These conditions must be met:
+    - If User has permissions to the Welcome Wizard Dashboard (merlin)
+    - On the Nautobot Home Page
+    """
+    content = format_html(
+        f"<a href={reverse('plugins:welcome_wizard:dashboard')}>The Nautobot Welcome Wizard can help you get started with Nautobot!</a>",
+        context.request.user,
```

Comment:
I don't see where this parameter is being used in the HTML format string - is it redundant?

Reply:
Good catch! I had it originally welcome the user, I will correct this.